src/foremast/utils/lookups.py

```diff
@@ -83,16 +92,16 @@ class FileLookup():
     """
 
     def __init__(self, git_short='', runway_dir=''):
+        """Init."""
```

Comment:
We are standardizing on docstrings on the Class. No need to add to `__init__`.

Reply:
pydoc is yelling at me. :)

Reply:
Fixed this too.